muxpi/set-up-image.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -e
+
+#If you want to run the neptune image, run `./set-up-image.sh neptune`
+NEPTUNE_IMAGE="$1"
+
+HOME="/home/muxpi"
+IMAGES="${HOME}/images"
+
+#The url for the latest successful nightly build.
+URL_MINIMAL="https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY/lastSuccessfulBuild/artifact/artifacts_rpi/core-image-pelux-minimal-dev-raspberrypi3.rpi-sdimg"
+VARIANT_MINIMAL="core-image-pelux-minimal-dev-raspberrypi3.rpi-sdimg"
+
+URL_NEPTUNE="https://pelux.io/jenkins/job/pelux-manifests_NIGHTLY/lastSuccessfulBuild/artifact/artifacts_rpi-qtauto/core-image-pelux-qtauto-neptune-dev-raspberrypi3.rpi-sdimg"
+VARIANT_NEPTUNE="core-image-pelux-qtauto-neptune-dev-raspberrypi3.rpi-sdimg"
+
+VARIANT=""
+URL=""
+if [ "$NEPTUNE_IMAGE" == neptune ]; then
+   VARIANT=$VARIANT_NEPTUNE
+   URL=$URL_NEPTUNE
+else 
+   URL=$URL_MINIMAL
+   VARIANT=$VARIANT_MINIMAL
```

Comment:
Why are they in different order? (Just a pedantic comment, but looks nicer if the same order is kept :smile: )